bucket_4D/xlockmore/specification

```diff
@@ -0,0 +1,53 @@
+DEF[PORTVERSION]=	5.56
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		xlockmore
+VERSION=		${PORTVERSION}
+KEYWORDS=		x11
+VARIANTS=		standard
+SDESC[standard]=	X Window System Lock Screen
+HOMEPAGE=		http://www.sillycycle.com/xlockmore.html
+CONTACT=		Leonid_Bobrov[mazocomp@disroot.org]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		http://sillycycle.com/xlock/
+DISTFILE[1]=		xlockmore-${PORTVERSION}.tar.xz:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		ISCL:single
+LICENSE_FILE=		ISCL:{{WRKDIR}}/LICENSE
+LICENSE_AWK=		ISCL:"^\#include"
+LICENSE_SOURCE=		ISCL:{{WRKSRC}}/xlock/xlock.c
+LICENSE_SCHEME=		solo
+
+FPC_EQUIVALENT=		x11/xlockmore
+
+USES=			c++:single
+
+XORG_COMPONENTS=	x11 xaw ice sm xpm xext xinerama
+
+BUILDRUN_DEPENDS=	freetype:primary:standard
+
+MUST_CONFIGURE=		gnu
+
+CONFIGURE_ARGS=		--without-motif --without-ttf --without-editres
+			--without-dtsaver --without-rplay --without-nas
+			--without-gtk --without-esound --disable-setuid
+			--without-mesa --without-opengl --without-ftgl
+			--without-gtk2 --without-magick --enable-blank-only
+			--enable-customization --disable-bomb
```

Comment:
Convention is that each one of these is a separate line.  You've got 3 per line which is legal but on the buildsheet they get converted to 1 per line anyway, so it's easy for humans to read that way as well.  Can you do that?  I know it makes file much longer but that's okay.

Reply:
Sure I can, furthermore I'll sort that alphabetically.